index.js

```diff
@@ -4702,9 +4703,35 @@ export async function createCsharpBom(path, options) {
         pkgData = pkgData.slice(1);
       }
       const dlist = parseCsPkgData(pkgData);
+      const deps = dlist;
       if (dlist?.length) {
         pkgList = pkgList.concat(dlist);
       }
+
+      if (dlist?.length) {
+        for (const p of dlist) {
+          parentDependsOn.add(p);
+        }
+      }
+    }
+    if (parentDependsOn.size) {
+      const arrrr =  new Set();
+      const prefix = parentComponent["bom-ref"].split("/")[0];
```

Comment:
If prefix is just `pkg:nuget` we can directly used that in 4723.

Reply:
For some reason this without this its not able to create the dependencies. I checked multiple times for this.